sos/report/plugins/tftpserver.py

```diff
@@ -21,8 +21,6 @@ class TftpServer(Plugin, IndependentPlugin):
     packages = ('tftp-server',)
 
     def setup(self):
-        self.add_cmd_output("ls -lanR /tftpboot")
-        self.add_cmd_output('ls -lanR /srv/tftp')
-
+        self.add_dir_listing(['/tftpboot', '/sr/tftp'], recursive=True)
```

Comment:
Also removing `-n` option, again no idea if it matters or not. I am ok with removal, if somebody will miss it, we can add it back.